selection_sort.go

```diff
@@ -0,0 +1,16 @@
+package sort
+
+func selectionSort(arr []int) []int {
+	for i := 0; i < len(arr)-1; i++ {
```

Comment:
The loop condition `i < len(arr)-1` is correct, but it's good practice to add a comment explaining why the loop runs until `len(arr)-1` instead of `len(arr)`.